### PR TITLE
Add CloudRun service methods

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/cloudprovider/cloudrun/cloudrun.go [hunk 1]
```diff
@@ -126,6 +126,26 @@ func (s *Service) ServiceManifest() (ServiceManifest, error) {
...
 }
 
+func (s *Service) UID() string {
+	if s.Metadata == nil {
+		return ""
+	}
+	return s.Metadata.Uid
+}
+
+// RevisionNames return some revision names which may handle traffic.
```
Comment (nghialv) [inline pkg/app/piped/cloudprovider/cloudrun/cloudrun.go:136]:
```suggestion
// RevisionNames returns all its active revisions which may handle the traffic.
```

File: pkg/app/piped/cloudprovider/cloudrun/cloudrun.go [hunk 2]
```diff
@@ -126,6 +126,26 @@ func (s *Service) ServiceManifest() (ServiceManifest, error) {
...
+func (s *Service) RevisionNames() []string {
```
Comment (nghialv) [inline pkg/app/piped/cloudprovider/cloudrun/cloudrun.go:137]: How about naming it with "Active" to narrow down the list?

```suggestion
func (s *Service) ActiveRevisionNames() []string {
```

File: pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go [hunk 3]
```diff
@@ -68,6 +68,14 @@ func (m ServiceManifest) Labels() map[string]string {
 	return m.u.GetLabels()
 }
 
+func (m ServiceManifest) AppID() string {
```
Comment (nghialv) [inline pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go:71]: Do we have any chance of missing the application ID in the service manifest?
If have, we need to return (string, bool).

Comment (knanao) [reply pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go:71]: Exactly, let me fix above.

**Comment by nghialv** [comment]
Nice.
/lgtm

**Comment by khanhtc1202** [comment]
Here you go
/approve

**Pull request merged**